src/python_eval_client.js

```diff
@@ -123,20 +123,11 @@ function evalDirectoryImp(directoryPath, cntkModelPath, cntkEnvDirPath, jsonTemp
     });
 }
 
-function EvalClient(cntkModelPath, cntkInstallDir, cntkEnv, verbose) {
+function EvalClient(cntkModelPath, cntkInstallDir, anacondaInstallDir, cntkEnv, verbose) {
     this.cntkInstallDir = cntkInstallDir;
     this.cntkModelPath = cntkModelPath;
     this.verbose = !!verbose;
 
-    var isWindows = process.platform == 'win32';
```

Comment:
This change was unecesarry, you could just add anacondaInstallDir as a parameter (after cntkEnv in the order) and then follow the code path you removed here in case it is null/undefined.
In general - less code changes means less bugs.